### ISSUE TITLE
NTR: CCL3 high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8440]
New term request CCL3 high in blood (elthat_220717052527): C-C motif chemokine ligand 3

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000687
- label: Elevated circulating CCL3 concentration
- parents: [Abnormal circulating CC chemokine concentration (HP:6000446)](https://hpo.jax.org/app/browse/term/HP:6000446)
- pmid: 
- def: The concentration of C-C Motif Ligand 3 (CCL3) in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8440
- synonyms:
